Makefile

```diff
@@ -11,12 +11,16 @@ test:
 	@ go test -timeout 10s ./...
 
 coverage: install-coverage
-	@coverage --keep-coverage-out --exclude="., mocks/*, pkg/backend/aws/testbackend/mock_*, pkg/stack_mgr/mock_*, pkg/workspace_repo/mock_*, pkg/backend/aws/interfaces/*" --covermode=atomic --coverprofile=coverage.txt --enforce
+	@coverage --keep-coverage-out --exclude="., *mock*, pkg/backend/aws/interfaces/*" --covermode=atomic --coverprofile=coverage.txt --enforce
```

Comment:
`coverage` tool doesn't allow individual file exclusion, so expression like `pkg/workspace_repo/mock_*` is ignored.